src/tools/interop/idt/scripts/activate.sh

```diff
@@ -0,0 +1,25 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+sudo docker run \
+    -it \
+    --mount source="$PWD"/idt,target=/idt,type=bind \
+    --privileged \
+    -v /dev/bus/usb:/dev/bus/usb \
+    -v /var/run/dbus:/var/run/dbus \
+    --net=host \
+    idt
```

Comment:
If might be nice for the script to check if the image has been built or not and maybe combine the build.sh and activate.sh together, maybe also the bootstrap.sh as well.

Reply:
We'll get to this!